R/bootstrap.nlsfit.R

```diff
@@ -639,10 +666,11 @@ bootstrap.nlsfit <- function(fn,
                              use.minpack.lm = TRUE,
                              parallel = FALSE,
                              error = sd,
-                             cov_fn = cov,
+                             cov.fn = cov,
```

Comment:
see question above regarding the change of notation here

If anything, `error` should become `err_fn` to increase consistency with the rest of the code.